plans/example/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG build_image
+ARG run_image
+
+FROM ${build_image} AS builder
+WORKDIR /build
+COPY . .
+RUN cd plan && go build -o /testplan 
+
+FROM ${run_image}
+COPY --from=builder /testplan /testplan
+COPY plan/artifact.txt /
```

Comment:
Including additional files in the image. Notice the path is "plan/artifact.txt", because the base context is one directory higher.

Reply:
Why is the base context one directory higher? I am not sure I understand this.